ai_genomics/pipeline/patent_data/get_ai_genomics_patents.py

```diff
@@ -0,0 +1,152 @@
+"""Script to query google patent data with USPTO
+AI patent ids and identify patent ids that have
+genomics-related classification codes.
+
+To run script,
+export GOOGLE_APPLICATION_CREDENTIALS="/path/to/credentials.json"
+python get_ai_genomics_patents.py
+"""
+#################################################
+from ai_genomics import bucket_name, config
+from ai_genomics.utils.patent_data.get_ai_genomics_patents_utils import est_conn
+from ai_genomics.getters.data_getters import (
+    s3,
+    load_s3_data,
+    save_to_s3,
+    get_s3_dir_files,
+)
+
+import pandas as pd
+import numpy as np
+
+from google.api_core.exceptions import Forbidden
+import uuid
+
+#################################################
+
+
+def make_query_chunks(uspto_patent_ids, n_chunks: int, table: str) -> list:
+    """Generate BigQuery query chunks based on USPTO AI patent IDS.
+
+    Args:
+        uspto_patent_ids: List of patent IDs.
+        n_chunks (int): The number of chunks generated from uspto_patent_ids.
+                        Chunk string size must be less than 1024.00K characters,
+                        including comments and white space characters. This means
+                        there needs to be approx. 85 query chunks to accomodate
+                        string size limits.
```

Comment:
Check out the formatting specified in the [google styleguide](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) (our preferred). Lines after the first should be one tab in. No types are needed if using type hints above.